CIP-0PRS/CIP-0PRS.lagda.md

```diff
@@ -0,0 +1,1355 @@
+---
+Title: Peras - Faster Settlement on Cardano
+CIP: ?
+Category: Consensus
```

Comment:
We don't have a `Consensus` category, but are there stakeholders & representatives that could _give_ us one?  It would be forward-looking to add it, and especially useful if there are more CIPs coming from you guys 🤓 but first this category would need to be "enlisted" through dedicated industry contacts as described for the currently enlisted categories here (`Ledger`, `Plutus`. `Catalyst`):
- https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#categories
- https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#project-enlisting

If you think `Consensus` meets the criteria for enlistment (based on commitment to reviews & projects following the CIP process) then I would be happy to have it.  We haven't enlisted any new categories since @KtorZ was an active editor but if you're interested we'll reciprocate as best we can (cc @Ryun1 @Crypto2099).

Reply:
@rphair, My understanding is that a separate CIP will create a "Consensus" category. We discussed this at a recent Consensus Working Group meeting.

Reply:
@bwbush it's good to hear that as a recommended strategy, since then likewise the first authors of `Consensus` CIPs would be the "enlisters" and effectively volunteering for review and advice about implementation.  cc @abailly-iohk

This is the best strategy I can think of to keep the CIP process decentralised with respect to supporting companies.  As this CIP moves forward through initial review I'll add `Consensus` to the official categories in CIP-0001.  cc @Ryun1 @Crypto2099

Reply:
Resolved with adding `Consensus` category in https://github.com/cardano-foundation/CIPs/pull/924.

@dnadales it still would be nice to get a CIP with some guidelines for `Consensus` CIPs as per https://github.com/cardano-foundation/CIPs/pull/872#discussion_r1706965548 & to promote it somewhat from where it currently exists here ("tentatively enlisted"): https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#categories (cc @bwbush & @Ryun1 @Crypto2099 @perturbing)